sw/device/lib/runtime/otbn.c

```diff
@@ -63,11 +63,11 @@ otbn_result_t otbn_busy_wait_for_done(otbn_t *ctx) {
     }
   }
 
-  dif_otbn_err_code_t err_code;
-  if (dif_otbn_get_err_code(&ctx->dif, &err_code) != kDifOtbnOk) {
+  dif_otbn_err_bitfield_t err_bits;
+  if (dif_otbn_get_err_bits(&ctx->dif, &err_bits) != kDifOtbnOk) {
     return kOtbnError;
   }
-  if (err_code != kDifOtbnErrCodeNoError) {
+  if (err_bits != 0) {
```

Comment:
How about adding a `uint32_t kDifOtbnErrBitNoError = 0` constant to the DIF and use that for the comparison?

Reply:
We don't allow the use of `static` in DIF headers and `const uint32_t kDifOtbnErrBitNoError = 0;` will result in multiple definitions of the symbol, so you have to do a `#define ...` or use an enum.

We could use a `#define` but there's no enum for it to go in (as the error bit enum gives bit indices and cannot express no bit set).

Reply:
... an argument for Philipp's suggestion of storing `1 << bit` rather than `bit`!